orc8r/cloud/go/services/configurator/client_api.go

```diff
@@ -320,96 +311,84 @@ func WriteEntities(networkID string, writes ...EntityWriteOperation) error {
 	return nil
 }
 
-func CreateEntity(networkID string, entity NetworkEntity) (NetworkEntity, error) {
-	ret, err := CreateEntities(networkID, []NetworkEntity{entity})
+func CreateEntity(networkID string, entity NetworkEntity) (SerializedNetworkEntity, error) {
+	ret, err := CreateEntities(networkID, NetworkEntities{entity})
 	if err != nil {
-		return NetworkEntity{}, err
+		return SerializedNetworkEntity{}, err
 	}
 	return ret[0], nil
 }
 
 // CreateEntities registers the given entities and returns the created network entities
-func CreateEntities(networkID string, entities []NetworkEntity) ([]NetworkEntity, error) {
+func CreateEntities(networkID string, entities NetworkEntities) (SerializedNetworkEntities, error) {
```

Comment:
It doesn't look like any callers actually use the first return value from this or UpdateEntities so maybe this is a good time to refactor it out. I don't like `SerializedNetworkEntity` as it is right now... let's just sidestep this problem for now.

A reasonable alternative would be to update this function signature to accept a serde registry that you can use on the RPC path when serializing the input and the return path when deserializing the RPC response.

Reply:
From offline: will add functionality to indicate whether a NetworkEntity has been deserialized

Reply:
Edit: I was getting (Serialized)NetworkEntity confused with (Serialized)State. So yes this is a new addition, and it's not needed internally, so yes we can remove it entirely